acceptance/tests/pxp-module-puppet/run_puppet_during_puppet.rb

```diff
@@ -28,9 +28,11 @@
   end
 
   step 'Start long-running Puppet agent jobs' do
-    agents.each do |agent|
-      on agent, puppet('agent', '--test', '--environment', environment_name, '--server', "#{master}",
-                       '</dev/null >/dev/null 2>&1 & echo $!')
+    with_puppet_running_on(master, { 'master' => { 'server' => master.fqdn }}) do
```

Comment:
Does this update with SIGHUP? Full restarts would slow tests a bit.

Reply:
Actually, this might not even be necessary -- I'll see how things work without it